docroot/themes/custom/vagovadmin/assets/scss/styles.scss

```
@@ -165,6 +165,28 @@
   }
 }
 
+.downloadable-file-link::before {
+  background-repeat: no-repeat;
+  background-size: 16px 16px;
+  content: '';
+  display: inline-block;
+  height: 16px;
+  padding-right: 4px;
+  width: 16px;
+}
```

```suggestion
.downloadable-file-link {
  padding-left: 20px;
  position: relative;

  &::before {
    background-repeat: no-repeat;
    background-size: 16px 16px;
    content: '';
    display: inline-block;
    height: 16px;
    padding-right: 4px;
    width: 16px;
  }
}
```

couldn't help to make it pixel perfect..